### ISSUE TITLE
new GO term requests: Host Cell Inclusion Body

### ISSUE THREAD

**Issue by djow2019** [geneontology/go-ontology#15580]
I would like to request the creation of three new GO terms which follow the Host Cell hierarchy. They are as follows:

Host Cell Inclusion Body
- Definition: A discrete intracellular part formed of aggregated molecules such as proteins or other biopolymers
- Source: PMID:11121744
- Position in Hierarchy: Is A Host Intracellular Part GO:0033646
- Synonyms: Host Inclusion Body

Host Cell Inclusion Membrane
- Definition: The lipid bilayer surrounding a host cell inclusion body
- Source: PMID:26416906
- Position in Hierarchy: Part of Host Cell Inclusion Body
- Synonyms: Host Inclusion Membrane

Host Cell Inclusion Lumen
- Definition: The volume enclosed by the membrane of the host cell inclusion body
- Source: PMID:18591235
- Position in Hierarchy: Part of Host Cell Inclusion Body
- Synonyms: Host Inclusion Lumen

**Comment by ukemi** [comment]
Could you please double-check these terms. According to PMID:26416906, the inclusion is a type of host vacuole, not necessarily an aggregate (?). Would it be better to make a vacuole term with a synonym and then make children of it?

**Comment by andrewsu** [comment]
@pgaudet we are interested in annotating localization of chlamydia proteins. So we considered [symbiont-containing vacuolar space (GO:0020004)](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0020004) (or one of its children), but I don't think this would be right from the perspective of Chlamydia proteins (right?).  Another possible candidate would be [host symbiosome (GO:0043658)](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0043658) (or one of its children).  Hmm, that one also has the advantage of already having children for [host peribacteroid fluid (GO:0043665)](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0043665) and [host peribacteroid membrane (GO:0043664)](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0043664), which roughly align with the child terms requested above.

@ukemi I asked @djow2019 to create the proposed definition for `host inclusion body` based on the existing term for [inclusion body (GO:0016234)](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0016234), but in retrospect perhaps that wasn't the best idea...

Basically we've been all over the map in terms of new terms vs existing terms, and how confident we are in that assessment.  At this point, some professional guidance would be much appreciated... ;)

Also tagging https://github.com/geneontology/helpdesk/issues/113 for completeness, and our domain Chlamydia domain experts @khybiske and @putmantime...

**Comment by pgaudet** [comment]
Thanks @andrewsu 
I've added all these contacts on our groups contact page: https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv

----
About the terms and the papers proposed here: 
* PMID:11121744 mentions inclusion bodies but from the perspective of what would be the host, and doesn't mention any symbiont, so we should not consider it as support for the requested terms. 

* PMID:26416906 described the 'inclusion' and en endosome that has been modified by the symbiont, so it's not a host structure, in my opinion (but it would be useful if  @khybiske and @putmantime would weigh in). 

My solution would be to create a term like 'symbiont inclusion body' that would be defined as a membrane-bounded intracellular organelle, originating by endocytosis of the symbiont (in the case of Chlamydia, this is referred to as elementary body (EB)), and then modified by the symbiont to avoid host defenses. 

It could be directly under 'GO:0005575 cellular_component', like 'GO:0039679 viral occlusion body' (or we could group these two terms as 'symbiotic structure found in host cell' or something along those lines). 

This are just quick suggestions, I am sure they can be improved. 

I am tagging @pmasson55 for his input on the similarities with viruses. 
 
Thanks, Pascale

**Comment by andrewsu** [comment]
Thank you @pgaudet and @ukemi .  I had a very informative conversation with @khybiske this afternoon, which I'll attempt to summarize here.  Kevin did not feel that "symbiont" was the best way to capture the organismal relationship given Chlamydia's role as a human pathogen.  He also educated me that the use of "inclusion" is perhaps a historical artifact in the Chlamydia community that doesn't generalize to analogous structures for other pathogens (e.g., TB, malaria, Salmonella).  So following @ukemi's suggestion, we converged on the following suggestion based on vacuole for your review:

**New term:** pathogen-containing vacuole
- description: intracellular compartment that is formed from internalization of a pathogen into a host cell
- alias: pathogen-containing compartment
- is_a: vacuole
- PMID: [10560000](https://www.ncbi.nlm.nih.gov/pubmed/10560000)
- has_part: pathogen-containing vacuole membrane
- has_part: pathogen-containing vacuole lumen

**New term:** pathogen-containing vacuole membrane
- description: host-derived membrane of a pathogen-containing vacuole
- PMID: [10560000](https://www.ncbi.nlm.nih.gov/pubmed/10560000)
- part_of: pathogen-containing vacuole

**New term:** pathogen-containing vacuole lumen
- description: lumen of a pathogen-containing vacuole
- PMID: [10560000](https://www.ncbi.nlm.nih.gov/pubmed/10560000)
- part_of: pathogen-containing vacuole

The above structure would also allow for children terms for pathogen-specific terms, for example:

**New term:** chlamydia inclusion
- description: intracellular compartment that is formed from internalization of Chlamydia into a host cell
- PMID: [12142274](https://www.ncbi.nlm.nih.gov/pubmed/12142274)
- is_a: pathogen-containing compartment

though if we did that, would we also need new terms for `chlamydia inclusion membrane` and `chlamydia inclusion lumen`?

Finally, Kevin clarified that `elementary body` and `reticulate body` are developmental forms of chlamydia that are found in (but not limited to) chlamydia inclusions.  We found matches in the [Brenda Tissue Ontology](http://obofoundry.org/ontology/bto.html) ([BTO_0000377](https://www.ebi.ac.uk/ols/ontologies/bto/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBTO_0000377) and [BTO_0001172](https://www.ebi.ac.uk/ols/ontologies/bto/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBTO_0001172), respectively). We defer to your judgement on whether these should be added to GO with an xref, or whether we should just use the BTO terms.

Thank you!

**Comment by pgaudet** [comment]
Hi @andrewsu 

The proposal looks reasonable to me, although 
(1) I don't understand why you need the term 'chlamydia inclusion'; do you envision to create distinct terms for any pathogen being integrated by this mechanism? Could we use the annotations to provide that information, ie if you are annotating a Chlamydia protein to 'pathogen-containing vacuole', aren't you implying that it's a chlamydia inclusion ? 

(2) For the ontology structure, as it is now, I think we need to specify whether we are describing a host structure. I don't know if 'vacuole' as a parent is ok. 
Thanks, Pascale

**Comment by andrewsu** [comment]
Hi @pgaudet,

(1) Thanks for raising this issue.  I forgot to mention that we are more than happy to defer the decision on whether to create the pathogen-specific term to your collective judgement.  You're right that we don't need it, and would be happy to annotate relative to the parent term of `pathogen-containing vacuole`. 

(2) Also very happy to defer to you here.  I think you're right that `vacuole` probably isn't correct, since that would imply it is a vacuole in the Chlamydia.  I wasn't sure about a parent in the `host cell` tree because the vacuole is created at the time of interaction between the host cell and the bacterium, so it's not a pre-existing structure within the host cell.  But having said all that, I trust your judgment on where to place these terms much more than mine...

thanks!

**Comment by pgaudet** [comment]
Thanks @andrewsu ! 
I would like to get @pmasson55 's input - (and anyone else who would like to comment). We'll make a suggestion tomorrow. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Talking with @pmasson55 and Catherine Rivoire, we propose this: 

host_cell part
-- pathogen inclusion 
---- bacterial inclusion 
---- viral occlusion body

-- pathogen inclusion membrane
---- bacterial inclusion membrane
---- (no viral occlusion body - not appropriate)

-- pathogen inclusion lumen
---- bacterial inclusion lumen
---- (no viral occlusion body- not appropriate)

OK with you @andrewsu @khybiske and @putmantime ? 

Thanks, Pascale

**Comment by ukemi** [comment]
Hi @pgaudet. I thought you said you would take this one.

**Comment by andrewsu** [comment]
Thank you!  I had another quick follow up conversation with @khybiske, and we agree that the structure you propose will serve our annotation needs just fine.  Having said that, we have two minor suggestions for your consideration.

- the use of "inclusion" appears to be somewhat specific to the chlamydia research community, and something more generic like "pathogen-containing vacuole" _may_ be more inclusive of other pathogen communities.

- if we interpret this correctly, you are proposing:
-- `host_cell` `has_part` `pathogen inclusion`
-- `host_cell` `has_part` `pathogen inclusion membrane`
-- `host_cell` `has_part` `pathogen inclusion lumen`
So that annotations to `pathogen inclusion membrane` have an explicit link to `pathogen inclusion` would this alternate structure make sense?
-- `host_cell` `has_part` `pathogen inclusion`
-- `pathogen inclusion` `has_part` `pathogen inclusion membrane`
-- `pathogen inclusion` `has_part` `pathogen inclusion lumen`

@khybiske please chime in to confirm I've summarized our discussion correctly, and @putmantime if there's anything from your perspective that we've missed.  Thank you everyone!

**Comment by pgaudet** [comment]
> if we interpret this correctly, you are proposing:
Yes @andrewsu , this is it. 

Thanks to all for the input, I'll create the terms.

**Comment by pgaudet** [comment]
Here are the terms. I edited the definitions a little, please  @andrewsu @khybiske  @putmantime let me know if it's OK. 


+[Term]
+id: GO:0140220
+name: pathogen-containing vacuole
+namespace: cellular_component
+def: "A membrane-bound intracellular compartment that is formed upon internalization of a pathogen into a host cell, and in which the pathogen resides." [PMID:10560000, PMID:26842840]
+synonym: "pathogen inclusion" RELATED []
+synonym: "pathogen-containing compartment" RELATED []
+is_a: GO:0033643 ! host cell part
+created_by: pg
+creation_date: 2018-04-20T06:02:14Z
+
+[Term]
+id: GO:0140221
+name: pathogen-containing vacuole membrane
+namespace: cellular_component
+def: "Host-derived membrane of a pathogen-containing vacuole." [PMID:10560000, PMID:26842840]
+synonym: "pathogen inclusion membrane" RELATED []
+synonym: "pathogen-containing compartment membrane" EXACT []
+is_a: GO:0016020 ! membrane
+relationship: part_of GO:0140220 ! pathogen-containing vacuole
+created_by: pg
+creation_date: 2018-04-20T06:18:25Z
+
+[Term]
+id: GO:0140222
+name: pathogen-containing vacuole lumen
+namespace: cellular_component
+def: "The enclosed volume within the sealed membrane of a pathogen-containing vacuole." [PMID:10560000, PMID:26842840]
+synonym: "lumen of a pathogen-containing vacuole" EXACT []
+synonym: "pathogen inclusion lumen" RELATED []
+synonym: "pathogen-containing compartment lumen" EXACT []
+synonym: "pathogen-containing vacuolar lumen" EXACT []
+is_a: GO:0031974 ! membrane-enclosed lumen
+relationship: part_of GO:0140220 ! pathogen-containing vacuole
+created_by: pg
+creation_date: 2018-04-20T06:24:02Z



Thanks, Pascale

**Comment by pgaudet** [comment]
@pmasson55 Is 'pathogen-containing vacuole' really the correct parent for 'viral occlusion body', given the definition above ? 

Thanks, Pascale

**Comment by andrewsu** [comment]
Looks great to me, thank you @pgaudet!
